RELEASE_NOTES.md

```diff
@@ -1,4 +1,9 @@
 # Release Notes
+## stable-1.15-1.3.0, stable-1.16-1.3.0
+
+* [ElasticSearch, fluentbit] Create index template
+  Create ElasticSearch Index Template. Require Fluentbit to deploy only after ElasticSearch deploys.
+
```

Comment:
Should we also include base tech bumps?

Reply:
The submitters of the PRs didn't think so.

Reply:
And a list of "bumped x" doesn't mean anything.